trident-use/gcp.adoc

```diff
@@ -63,6 +63,8 @@ See the following table for the backend configuration options:
 
 | `serviceLevel` |The CVS service level for new volumes. The values are "standard", "premium", and "extreme". |"standard"
 
+|`network` |GCP network used for CVS volumes, defaults to “default” |“default”
```

Comment:
```defaults to "default"``` seems kinda redundant with the subsequent column.  Otherwise, looks great.

Reply:
Removed it :-)